Tools/AP_Periph/can.cpp

```diff
@@ -1096,8 +1144,10 @@ static void process1HzTasks(uint64_t timestamp_usec)
          * The recommended way to establish the minimal size of the memory pool is to stress-test the application and
          * record the worst case memory usage.
          */
-        if (pool_peak_percent() > 70) {
-            printf("WARNING: ENLARGE MEMORY POOL\n");
+        for (uint8_t iface = 0; iface < HAL_NUM_CAN_IFACES; iface++) {
+            if (pool_peak_percent(iface) > 70) {
+                printf("WARNING: ENLARGE MEMORY POOL on Iface %d\n", iface);
```

Comment:
doesn't this need a (int)iface   ?

Reply:
looks like not, otherwise would have failed to build